### PR TITLE
Page migration

### PR DESCRIPTION
**Pull request by bottlerocket**
Moved across original build using the NHS framework to a new Jekyll Framework

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,38 +1,16 @@
...
-
-      <div class="footer-col footer-col-3">
-        <p>{{ site.description }}</p>
-      </div>
-    </div>
+<footer id="global-footer" role="contentinfo">
+  <div class="footer-container">
+    <nav>
+      <ul>
+        <li><a class="on-blue" href="/help/cookies">Cookies</a></li>
```
Comment (Demotive) [inline _includes/footer.html]: I'd suggest either deleting these links, or adding appropriate pages...

File: _includes/footer.html [hunk 2]
```diff
@@ -1,38 +1,16 @@
...
+        <li><a class="on-blue" href="/help/clinical-assurance">Clinical assurance</a></li>
+      </ul>
+    </nav>
+<!--
```
Comment (Demotive) [inline _includes/footer.html]: Delete rather than comment out

File: _includes/header.html [hunk 3]
```diff
@@ -1,27 +1,9 @@
...
-      <div class="trigger">
-        {% for my_page in site.pages %}
-          {% if my_page.title %}
-          <a class="page-link" href="{{ my_page.url | prepend: site.baseurl }}">{{ my_page.title }}</a>
-          {% endif %}
-        {% endfor %}
-      </div>
-    </nav>
+    <a href="/" class="header-logo" title="Go to the NHS.UK homepage">
+      <img src="/images/nhs-logo.svg" alt="NHS" width="82" height="32" /> <!--width="104" height="42" -->
```
Comment (Demotive) [inline _includes/header.html]: Delete rather than comment out ;)

**Comment by Demotive** [comment]
If you're not using the `work-panel-bottom-edge` images, I'd remove them.

**Comment by Demotive** [comment]
Have you thought about a `:focus` style?

**Comment by Demotive** [comment]
Should the `<title>` reflect the `<h1>` - maybe "NHS Beta: transforming digital health and care"

File: _sass/modules/hero/_hero.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.hero {
+  background-color: $white;
```
Comment (Demotive) [inline _sass/modules/hero/_hero.scss]: This needs to be a blue - if there's no background image loaded, you get white text on white bg.

**Comment by Demotive** [comment]
Might be worth trimming down some of the unused scss (for example in `_colours.scss`) ?

File: _layouts/default.html [hunk 5]
```diff
@@ -7,11 +7,169 @@
...
+
+                    <div class="grid-row">
+
+                        <div class="grid-2-3 work-intro">
+
+                            <h2 class="header">The user at the heart of NHS.UK</h2>
+
+                            <p class="body-copy">Our work is research-led and focuses on the needs of patients, carers and front-line staff. We constantly iterate. Our work starts small, is tested and evolves based on users’ feedback. We make things open. We want to share, learn and adapt. We work in partnership with digital health and care teams and across organisational boundaries.</p>
+
+                            <p class="body-copy">We’ve made the following:</p>
```
Comment (Demotive) [inline _layouts/default.html]: Sounds weird, but perhaps this should be a heading for the following modules?

File: _layouts/default.html [hunk 6]
```diff
@@ -7,11 +7,169 @@
...
+
+      <div id="our-focus">
+
+          <div class="content">
+
+                <div class="grid-row">
+
+                    <div class="grid-2-3 focus-intro">
+                        <p>NHS.UK projects form the building blocks of a new digital health and care service. A service that can deliver the next generation of digital healthcare, help reduce pressure on the NHS, improve efficiency and increase quality for all.</p>
+                        <p>Our current focus:</p>
```
Comment (Demotive) [inline _layouts/default.html]: I think this should be a heading for the following area. Doesn't have to visually reflect that, but semantically it feels more that way?

File: _layouts/default.html [hunk 7]
```diff
@@ -7,11 +7,169 @@
...
+
+                </div>
+
+                <div class="grid-row">
+
+                    <div class="focus-panel">
+                        <div class="focus-number">1</div>
+                        <h3 class="focus-title">NHS.UK</h3>
+                        <div class="divider"></div>
+                        <div class="focus-description">Transforming NHS Choices to deliver the next generation of digital services with relevant, practical, action-focused content and smart sign-posting to services.</div>
```
Comment (Demotive) [inline _layouts/default.html]: Should these be `<p>` paras?

File: _layouts/default.html [hunk 8]
```diff
@@ -7,11 +7,169 @@
...
+                        <h3 class="focus-title">Register with a GP</h3>
+                        <div class="divider"></div>
+                        <div class="focus-description">An exemplar digital service designed to simplify the process for patients and help to reduce pressure on key NHS services.</div>
+                    </div>  
+
+                    <div class="focus-panel">
+                        <div class="focus-number">4</div>
+                        <h3 class="focus-title">Partnerships</h3>
+                        <div class="divider"></div>
+                        <div class="focus-description">Working in partnership with digital healthcare teams, across organisational boundaries, to drive quality and improve digital take-up.</div> <!-- ed: I hate language like this -->
```
Comment (Demotive) [inline _layouts/default.html]: Probably delete that stray comment ;)

File: _layouts/default.html [hunk 9]
```diff
@@ -7,11 +7,169 @@
...
+      <div class="work-module -hero">
+          <!-- <div class="edge"></div> -->
+          <div class="content">
+
+              <div class="grid-row">
+
+                  <div class="grid-12">
+
+                        <div class="module--lead">
+
```
Comment (Demotive) [inline _layouts/default.html]: Maybe try something like:

```
<h2 class="module--header">
  <span class="module--type">Long term conditions</span>
  I need to manage a long-term condition
</h2>
```

I'd avoid preceding an `<h2>` with an `<h4>` like that...

**Comment by Demotive** [comment]
Make sure you declare the language in the html page - see https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_html_01

File: _includes/header.html (hunk 3 above)
Comment (Demotive) [inline _includes/header.html]: Also if we're going "production" it's probably worth swapping the logo .svg for an @2x.png or something.

File: _includes/header.html [hunk 10]
```diff
@@ -1,27 +1,9 @@
...
-
-      <div class="trigger">
-        {% for my_page in site.pages %}
-          {% if my_page.title %}
-          <a class="page-link" href="{{ my_page.url | prepend: site.baseurl }}">{{ my_page.title }}</a>
-          {% endif %}
-        {% endfor %}
-      </div>
-    </nav>
+    <a href="/" class="header-logo" title="Go to the NHS.UK homepage">
```
Comment (Demotive) [inline _includes/header.html]: Change the `title` attribute.

In fact, does the NHS logo even *need* to be a link in this page?

File: _layouts/default.html [hunk 11]
```diff
@@ -7,11 +7,169 @@
...
+                        <h3 class="focus-title">Digital service standards</h3>
+                        <div class="divider"></div>
+                        <div class="focus-description ">Publishing service standards and practical guidance to help teams deliver quality, safe and trusted digital services.</div>
+                    </div>
+                    
+                    <div class="focus-panel">
+                        <div class="focus-number">3</div>
+                        <h3 class="focus-title">Register with a GP</h3>
+                        <div class="divider"></div>
+                        <div class="focus-description">An exemplar digital service designed to simplify the process for patients and help to reduce pressure on key NHS services.</div>
```
Comment (Demotive) [inline _layouts/default.html]: `<p>` ?

File: _layouts/default.html (hunk 8 above)
Comment (Demotive) [inline _layouts/default.html]: `<p>` ?

File: _config.yml [hunk 12]
```diff
@@ -6,16 +6,16 @@
...
 email: your-email@domain.com
 description: > # this means to ignore newlines until "baseurl:"
   Write an awesome description for your new site here. You can edit this
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "http://yourdomain.com" # the base hostname & protocol for your site
-twitter_username: jekyllrb
-github_username:  jekyll
+twitter_username:
```
Comment (Demotive) [inline _config.yml]: If you're not using these, I think you may as well delete them...

File: _includes/header.html [hunk 13]
```diff
@@ -1,27 +1,9 @@
-<header class="site-header">
+<header role="banner" id="global-header">
```
Comment (Demotive) [inline _includes/header.html]: I'd strip out stray `id` attributes if we're not actively using them?

File: _includes/footer.html (hunk 1 above)
Comment (bottlerocket) [inline _includes/footer.html]: What's the situation with the current footer? And the isseus aroun OGL, etc. We should replicate what you did there.

File: _includes/header.html (hunk 3 above)
Comment (bottlerocket) [inline _includes/header.html]: Will add a ticket for that

File: _layouts/default.html [hunk 14]
```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<html lang="en-GB">
 <html>
```
Comment (Demotive) [inline _layouts/default.html]: Get rid of the extra `html` tag

**Pull request merged**